.github/workflows/nightly.yaml

```
@@ -6,8 +6,8 @@ on:
     - cron: 0 0 * * 1-5
 
 env:
-  DOCKER_REGISTRY: quay.io
-  DOCKER_REPOSITORY: dynatrace/dynatrace-operator
+  DOCKER_REGISTRY: ghcr.io
```

tiiiny nitpick, feel free to ignore

```suggestion
  IMAGE_REGISTRY: ghcr.io
```